applications/fluid_structure_interaction/perpendicular_flap/fluid-exadg/input.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
In general not. It's more a matter of taste: right now each solver is started from a subdirectory (this one) so that all log files and output files are clearly separated. Of course, we could also execute everything from the same (root) directory, but it becomes in my opinion rather messy.